src/layout/RootLayout.tsx

```diff
@@ -10,6 +10,10 @@ import '@/css/globals.css';
 import { Auth0Provider } from '@/lib/auth0-provider';
 import { isProd, isStaging } from '@/helpers/env';
 
+if (process.env.NODE_ENV === 'development') {
+  Settings.throwOnInvalid = true;
+}
```

Comment:
annoying to have silent failures on wrong dates while developing and will also help us catch bugs